synapse/models/syn.py

```diff
@@ -20,8 +20,11 @@ def _normPyStr(self, text):
                 return user.iden, {}
 
             user = core.auth._getUserByName(text)
-            if user is not None:
-                return user.iden, {}
+            if user is None:
+                mesg = f'No user named {text}.'
+                raise s_exc.NoSuchUser(mesg=mesg, username=text)
```

Comment:
This can lead to the creation of nodes that cannot be merged with `merge --apply`. We can take a different approach of adding a try/except around the `Guid.__normPyStr` in order to  make it clear to a caller that the user does not exist and the value isn't a valid guid property.

Reply:
Fixed, thanks for the help diagnosing this.